rest_witchcraft/serializers.py

```diff
@@ -922,13 +922,9 @@ def _expandable_fields(self):
         root = self.root
         f = self
         while f is not root:
-            if f.parent is root and isinstance(f.parent, serializers.ListSerializer):
```

Comment:
not sure what was smoking when did this but its broken. does not account for listserializer multiple levels deep. logic should be pretty simple. if field name exists (e.g. on regular serializer or list serializer itself), add to components else ignore (e.g. list child serializer)